core/module/included_spec.rb

```diff
@@ -36,4 +36,8 @@ def test
   it "is private in its default implementation" do
     Module.should have_private_instance_method(:included)
   end
+
+  it "works with super using a singleton class" do
+    ModuleSpecs::SingletonOnModuleCase::Bar.included_called?.should == true
```

Comment:
I'll do the fix of moving the include in the spec example, thanks for all the new super specs :smiley: